spec/requests/api/graphql/queries/submissions_spec.rb

```diff
@@ -87,6 +87,28 @@
         expect(error_message).to eq "Can't load other people's submissions."
       end
     end
+
+    context 'with a request from a partner and wrong submission id' do
```

Comment:
This test ensures that a partner who does not own the submission (`submission.user_id != partner.user_id`) cannot access the permission by querying for it by ID.